package.json

```diff
@@ -5,8 +5,8 @@
   "name": "design-system-prototypes",
   "private": true,
   "engines": {
-    "node": "^20",
-    "npm": "^10"
+    "node": "20",
+    "npm": "10"
   },
   "volta": {
```

Comment:
Maybe we can get rid of this as well? I think Ruben doesn't use it anymore

Reply:
Sure.